frontend/src/metabase/common/components/FilterPicker/DateFilterPicker/DateFilterPicker.tsx

```diff
@@ -58,3 +61,45 @@ export function DateFilterPicker({
     </div>
   );
 }
+
+interface SimpleDateFilterPickerProps {
+  query: Lib.Query;
+  stageIndex: number;
+  column: Lib.ColumnMetadata;
+  filter?: Lib.FilterClause;
+  onChange: (filter: Lib.ExpressionClause | undefined) => void;
+}
+
+export function SimpleDateFilterPicker({
+  query,
+  stageIndex,
+  column,
+  filter,
+  onChange,
+}: SimpleDateFilterPickerProps) {
```

Comment:
nit: I think it'd be more intuitive to have a `SimpleDateFilterPicker` file

Reply:
Agree, fixed